tests/perf/run_perf_tests.py

```diff
@@ -7,69 +7,80 @@
 import os
 import csv
 import sys
+import getopt
 from datetime import datetime
 
 DEFAULT_TEST_SEQUENCE = "50:30,200:30,200:60,400:30"
 DEFAULT_REPETITIONS = 10
-DEFAULT_VEGETA_PATTERN_TAR_FILE = ""
-DEFAULT_DAEMON_ON_CORE = "-"
-DEFAULT_GETH_HOME_DIR = "../../../turbo-geth/"
+DEFAULT_VEGETA_PATTERN_TAR_FILE = "turbo_geth_stress_test_001.tar"
```

Comment:
I would change this default to something like `./vegeta/turbo_geth_stress_test_001.tar` and put the file under Git control

Reply:
ok